SecuritySystemUWP/SecuritySystemUWP/AllJoynManager.cs

```diff
@@ -55,7 +55,8 @@ class AllJoynManager : ISecuritySystemService
             var images = await this.dropFolder.GetFilesAsync();
             var orderedImages = images.OrderByDescending(x => x.DateCreated);
             var file = orderedImages.FirstOrDefault();
-            if (null != file && file.Name != this.newestImage.Name)
+
+            if ((null == this.newestImage) || (null != file && file.Name != this.newestImage.Name))
             {
                 this.newestImage = file;
```

Comment:
As a future reference: the current change won't crash, because even though newest image can be set to null, this handles updating a null newest image anyway. I'm changing this so that newestImage will never reset to null, since once there has ever been a newest image, it should only ever not be the newest image when a newer image comes along.